src/utilities/geometry/Polyhedron.cpp

```diff
@@ -118,10 +118,23 @@ Surface3d::Surface3d(std::vector<Point3d> t_vertices, std::string t_name, size_t
       itnext = std::begin(vertices);
     }
 
-    edges.emplace_back(*it, *itnext, t_name, t_surfNum);
+    edges.emplace_back(*it, *itnext, name, surfNum);
```

Comment:
Fix an unrelated issue, t_name (the parameter) was moved from, so it's generally an empty string.